contentcuration/contentcuration/models.py

```diff
@@ -268,23 +268,24 @@ class ContentNode(MPTTModel, models.Model):
     objects = TreeManager()
 
     def save(self, *args, **kwargs):
-        isNew = self.pk is None
-
         # Detect if node has been moved to another tree
-        if not isNew and ContentNode.objects.filter(pk=self.pk).exists():
+        if self.pk is not None and ContentNode.objects.filter(pk=self.pk).exists():
             original = ContentNode.objects.get(pk=self.pk)
             if original.parent and original.parent_id != self.parent_id:
                 original.parent.changed = True
                 original.parent.save()
 
         super(ContentNode, self).save(*args, **kwargs)
-        if isNew:
-            self.original_node = self.pk if self.original_node is None else self.original_node
-            self.cloned_source = self.pk if self.cloned_source is None else self.cloned_source
+        post_save_changes = False
```

Comment:
Saves original/cloned source properly https://trello.com/c/60LZzI3r/192-when-importing-content-item-across-channels-some-metadata-should-be-locked-and-no-longer-be-editable-e-g-license-copyright-holde